docker/Dockerfile-ubuntu-20

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:20.04 as builder
```

Comment:
Looks like you use here Multi-stage builds, plz add info about in into PR description. 
Also @kotbegemot PTAL, as more I see cases like this more I want to use separate dockerfile for  C# buuut... We need to discuss it.